docker-compose.yml

```diff
@@ -29,8 +29,7 @@ x-build-volumes: &build-volumes
     - ${CCACHE_DIR:-container-ccache}:/root/ccache
 
 x-build-common-definitions: &build-common-definitions
-  <<: *common-definitions
-  <<: *build-volumes
+  <<: [*common-definitions, *build-volumes]
```

Comment:
These seem unnecessary formatting changes here and across the rest of this file. I would recommend backing those out and trying if your build works without them